files/pt-br/web/javascript/reference/global_objects/object/tostring/index.html

```diff
@@ -1,17 +1,27 @@
 ---
 title: Object.prototype.toString()
 slug: Web/JavaScript/Reference/Global_Objects/Object/toString
+tags:
+ - JavaScript
+ - Method
+ - Object
+ - Prototype
+browser-compat: javascript.builtins.Object.toString
 translation_of: Web/JavaScript/Reference/Global_Objects/Object/toString
 ---
 <div>{{JSRef("Global_Objects", "Object")}}</div>
 
 <h2 id="Summary" name="Summary">Sumário</h2>
 
-<p>O método <code><strong>toString()</strong></code> retorna uma string representando o objeto.</p>
+<p>O método <code>toString()</code> retorna uma string representando o objeto.</p>
+
+{{EmbedInteractiveExample("pages/js/object-prototype-tostring.html")}}
 
 <h2 id="Syntax" name="Syntax">Sintaxe</h2>
 
-<pre class="syntaxbox"><code><var>obj</var>.toString()</code></pre>
+<pre class="brush: js">
+toString()
+</pre>
```

Comment:
Obrigado pela contribuição!

Na alteração abaixo parece estar faltando o `obj.`, segue uma sugestão:

```suggestion
<pre class="brush: js">
toString()
</pre>
```